code/modules/client/client_procs.dm

```diff
@@ -166,6 +166,9 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 
 	if(CONFIG_GET(flag/automute_on) && !holder && last_message == message)
+		if(SEND_SIGNAL(src, COMSIG_CLIENT_AUTOMUTE_CHECK, last_message, mute_type) & SIGVAL_WAIVE_AUTOMUTE_CHECK)
```

Comment:
It might honestly be better to just do this on the mob so you don't have to do all that login stuff, but it's fine if you don't want to.

Reply:
That's probably easier. I think I'll try that.